### ISSUE TITLE
Issue with HK U+5D6B 嵫 and U+6ECB 滋

### ISSUE THREAD

**Issue by NightFurySL2001** [adobe-fonts/source-han-sans#486]
U+5D6B 嵫 is a Big5 level 2 character.
A new glyph is required for HK. It can then be mapped to KR following code chart (?).
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/227896c3-e61c-4197-baa1-f839a15a103d)
Affects Serif as well.
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/3f18d195-125c-442a-9ac0-4c88fae4f21a)

滋 U+6ECB requires a new HK glyph too. Same for Serif.
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/d9dc842a-41ce-4612-9264-f82d1997c49f)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/1adf1099-3d4c-42f1-935e-89bc311b6413)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/94c90345-7812-4946-a6cd-85537dc24b20)
In EDB:
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/536590e8-0399-4f27-8d7e-d683a740431b)

**Comment by hfhchan-plb** [comment]
I would suggest putting this character change on hold, because the Education Bureau of Hong Kong can't even decide on a consistent shape. See 孳:
![image](https://github.com/adobe-fonts/source-han-sans/assets/112690257/9536ce9b-c191-4c34-b66b-ef6bc00e5629)

vs 慈:
![image](https://github.com/adobe-fonts/source-han-sans/assets/112690257/407fb50c-8a6c-4bdc-aede-823ae92f3726)

Meanwhile, both Ping Fang HK and most commerically available fonts use 兹 instead of 茲.

**Comment by NightFurySL2001** [comment]
It seems that characters in Big5 that had a variant of 茲 in HKSCS shows up as 兹 in EDB. 

孳 U+5B73 (Big5) vs 孶 U+5B76 (HKSCS)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/67db3efc-fd59-4510-b217-e7b4e302d4bc)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/dee4aa78-4706-45a4-b64d-aaa7b4d86750)

慈 U+6148 (Big5) vs 慈 U+2F8A6 (HKSCS)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/1427b566-4083-4a79-a04b-45b30bf3594b)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/c2100047-c180-4fc3-a53f-5efbcee6b285)

磁 U+78C1 (Big5) vs 𥔵 U+25535 (HKSCS)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/3bbc6f5c-b5c8-4cb1-b6a8-adf00e1408a1)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/9c704c42-c745-45cc-b568-adb79739dbbe)

Reference Glyphs for Chinese Computer Systems in Hong Kong (2016) show U+5D6B 嵫 and 滋 U+6ECB used 兹. However in the latest code charts, these 2 glyphs have been changed to use 茲. This seem to be done in Unicode 13.0 without any IRG documents.
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/47fe1648-649a-45da-9310-96886dfbe3e3)
![image](https://github.com/adobe-fonts/source-han-sans/assets/33471049/6c6c2849-e1ca-479a-843b-e5bd0b7e45a7)


Otherwise, the characters (no matter Big5 or HKSCS) uses 茲, not 兹.

**Comment by hfhchan** [comment]
(I am reposting this because I realized I posted this under the wrong account. @NightFurySL2001 was responding to this post previously.)

I would suggest putting this character change on hold, because the Education Bureau of Hong Kong can't even decide on a consistent shape. See 孳:
![image](https://github.com/adobe-fonts/source-han-sans/assets/112690257/9536ce9b-c191-4c34-b66b-ef6bc00e5629)

vs 慈:
![image](https://github.com/adobe-fonts/source-han-sans/assets/112690257/407fb50c-8a6c-4bdc-aede-823ae92f3726)

Meanwhile, both Ping Fang HK and most commerically available fonts use 兹 instead of 茲.

**Comment by hfhchan** [comment]
Yes I am aware most glyphs for Reference Glyphs for Chinese Computer Systems in Hong Kong (2016) have been updated to use the grass head, but practically all the commerical fonts produced for Hong Kong do not use this shape.

I will be writing a document to CLIAC shortly after I gather all relevant materials and evidences, so I would advise Adobe to temporarily put this on hold before it is sorted out.
